scripts/policy_tag_extractor/README.md

```diff
@@ -0,0 +1,23 @@
+# BigQuery Policy Tag Extractor
+
+## Introduction
+This repo contains a bash script for extracting BigQuery policy tag information from a given dataset. The script will review all the tables in a dataset, and output the table name, column, name, and policy tag ID in a CSV format.
```

Comment:
```suggestion
This directory contains the [policy_tag_export.sh](policy_tag_extractor) bash script which extracts BigQuery policy tag information from a given dataset. The script will iterate through at most 10K tables in a dataset and then for every column with a policy tag, it will output the table name, column name, and policy tag ID in CSV format.
```